docs/src/installation.md

```diff
@@ -107,6 +107,7 @@ The link in the `Solver` column is the corresponding Julia package.
 | [CDD](https://github.com/cddlib/cddlib)                                        | [CDDLib.jl](https://github.com/JuliaPolyhedra/CDDLib.jl)                         |        | GPL      | LP                        |
 | [Clarabel.jl](https://github.com/oxfordcontrol/Clarabel.jl)                    |                                                                                  |        | Apache   | LP, QP, SOCP, SDP         |
 | [Clp](https://github.com/coin-or/Clp)                                          | [Clp.jl](https://github.com/jump-dev/Clp.jl)                                     |        | EPL      | LP                        |
+| [COPT](https://www.shanshu.ai/copt)                                            | [COPT.jl](https://github.com/COPT-Public/COPT.jl)                                |        | Comm.    | (MI)LP, (MI)SOCP, SDP     |
```

Comment:
Here:
https://github.com/COPT-Public/COPT.jl/blob/main/test/MathOptInterface/MOI_wrapper.jl#L40
it says that COPT does not support MISOCP